Update set-browser-ttl.md (#7814)
cloudflare/cloudflare-docs: content/cache/how-to/set-browser-ttl.md

3.  Under **Browser Cache TTL**, select the drop-down menu to select the desired cache expiration time.

The **Respect Existing Headers** option tells Cloudflare to honor the settings in the `Cache-Control` headers from your origin web server.

{{<Aside type="note" header="Respect Existing Headers Availability">}}

> For all plan types, you can set Browser Cache TTL to Respect Existing Headers at a zone level (zone setting). If you want to set Browser Cache TTL to Respect Existing Headers via Page Rules, this is currently only a Enterprise zone feature. 

Reviewer: ```suggestion
For all plan types, you can set Browser Cache TTL to Respect Existing Headers at a zone level (zone setting) or you can use a Cache Rule. Setting Browser Cache TTL to Respect Existing Headers via Page Rule is currently an Enterprise-only zone feature. 
```
